org.firo.firo-qt.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "org.firo.firo-qt",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "23.08",
+    "command": "firo-qt",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=/run/tor/control.authcookie:ro"
+    ],
+    "modules": [
+        {
+            "name": "firo",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -m 0755 -D -t /app/bin/internal bin/firod",
+                "install -m 0755 -D -t /app/bin/internal bin/firo-qt",
+                "install -m 0755 -D -t /app/bin/internal bin/firo-cli",
+                "desktop-file-edit --set-key=\"Icon\" --set-value=${FLATPAK_ID} firo-qt.desktop",
+                "install -Dp -m 644 firo-qt.desktop /app/share/applications/${FLATPAK_ID}.desktop",
+                "install -Dp -m 644 firo.svg /app/share/icons/hicolor/scalable/apps/org.firo.firo-qt.svg",
+                "install -Dp -m 644 org.firo.firo-qt.metainfo.xml /app/share/metainfo/org.firo.firo-qt.metainfo.xml",
+                "for b in firo{d,-qt,-cli}; do echo '#!/usr/bin/bash' > /app/bin/$b && echo \"exec /app/bin/internal/$b\" '-datadir=\"${XDG_DATA_HOME}\" \"$@\"' >> /app/bin/$b && chmod 744 /app/bin/$b; done"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/firoorg/firo/releases/download/v0.14.13.2/firo-0.14.13.2-linux64.tar.gz",
```

Comment:
so what happen in the aarch64 package?

Reply:
aarch64 support is not maintained, so i haven't added it. Maybe i'm missing the problem. Should we provide an aarch64 package anyway for flathub even if we don't support it upstream?

Reply:
You are building for both arches right now while shipping a x86_64 only binary https://docs.flathub.org/docs/for-app-authors/maintenance#limiting-the-set-of-architectures-to-build-on

Reply:
I missed that a flatpak.json file was also needed containing `only-arches` . Added.